clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -1,78 +1,3 @@
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: dptp-controller-manager
-  namespace: ci
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: dptp-controller-manager
-rules:
-- apiGroups:
-  - prow.k8s.io
-  resources:
-  - prowjobs
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  resourceNames:
-  - dptp-controller-manager
```

Comment:
This ClusterRole has to be split up because there is a generic part and a part that is specific for app.ci. Only on the latter it can:
* Manage Prowjobs
* Access the `dptp-controller-manager` configmap
* Create events
* Access `testimagestreamtagimports`

I would like to keep that split because it will show us programming errors